src/Tribe/Tickets.php

```diff
@@ -2262,6 +2263,148 @@ public function ticket_add( $post_id, $data ) {
 			return $save_ticket;
 		}
 
+		/**
+		 * Get the saved or default ticket provider
+		 *
+		 * @since 4.7
+		 *
+		 * @param int $event_id - the post id of the event the ticket is attached to.
+		 *
+		 * @return string ticket module class name
+		 */
+		public static function get_event_ticket_provider( $event_id = null ) {
+
+			// if  post ID is set, and a value has been saved, return the saved value
+			if ( ! empty( $event_id ) ) {
+				$saved = get_post_meta( $event_id, tribe( 'tickets.handler' )->key_provider_field, true );
+
+				if ( ! empty( $saved ) ) {
+					return $saved;
+				}
+			}
+
+			// otherwise just return the default
+			return self::get_default_module();
+		}
+
+		// @codingStandardsIgnoreEnd
+
+
+		/**
+		 * Parent method to be pass to any child of this class.
```

Comment:
This docblock is - incorrect? Or at least missing info on what the function does?